tests/default/_core/aggregation/nested.yaml

```
@@ -0,0 +1,100 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test aggregation endpoints with nested.
```

I'll nitpick, there's no such a thing as aggregation endpoint. 

```suggestion
description: Test aggregation.
```